pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1515,6 +1515,16 @@ func (og *operationGenerator) GenerateVerifyControllerAttachedVolumeFunc(
 		return volumetypes.GeneratedOperations{}, volumeToMount.GenerateErrorDetailed("VerifyControllerAttachedVolume.FindPluginBySpec failed", err)
 	}
 
+	if volumeToMount.PluginIsAttachable {
```

Comment:
Instead of changing the `VolumeHost` interface, would it be easier to pass Node object (or `volumesAttached`) here as an argument? Or do the check in the reconciler?

Reply:
I think underlying complexity of doing that via reconciler is almost the same. Passing something like `volumesAttached` to DSOW will require modifying DSOW populator each time it reconciles. This will be more or less one more reconcile loop that we will depend on and I do not like it (for various reasons). 

There is a "prior-art" of asking kubelet's VolumeHost for such stuff (look at node's allocatable) and I think it might be cleanest. It would have been nicer if we could query this directly in reconciler but we do not have `VolumeHost` interface access in reconciler but I think overhead of moving the check in `operation_generator` is minimal.

Reply:
Ack